code/modules/industrial_lift/tram/tram_machinery.dm

```diff
@@ -836,6 +858,41 @@ GLOBAL_LIST_EMPTY(tram_doors)
 	if(tram_part.travelling) //making a daring exit midtravel? make sure the doors don't go in the wrong state on arrival.
 		return PROCESS_KILL
 
+/obj/machinery/button/tram
+	name = "tram request"
+	desc = "A button for calling the tram. It has a speakerbox in it with some internals."
+	icon_state = "tramctrl"
+	skin = "tramctrl"
+	light_power = 0.5
+	light_range = 1.5
+	light_color = LIGHT_COLOR_DARK_BLUE
+	var/light_mask = "tram-light-mask"
+	device_type = /obj/item/assembly/control/tram
+	req_access = list()
+	id = 1
+	/// The specific lift id of the tram we're calling.
+	var/lift_id = MAIN_STATION_TRAM
```

Comment:
re-order your vars so that the new ones you define come after the ones it inherits. also dmdoc `light_mask` for coverage's sakes